index.js

```diff
@@ -17,9 +18,12 @@ module.exports = function (opts) {
 			return;
 		}
 
-		(new Vulcanize(opts)).process(file.path, function (err, inlinedHtml) {
+		// vulcanize expects target path to be relative to abspath
+		let filePath = opts.abspath ? path.relative(opts.abspath, file.path) : file.path;
```

Comment:
I'm not sure what you mean by "the convention is to support absolute paths," because vulcanize pushes the target path you give it through the same loader as it does everything else.

Their own documentation says that `abspath` should be "an absolute path to `target`," which isn't entirely clear to me, but does point to some expectations between `target` and `abspath`.

Furthermore, vulcanize has issues like Polymer/vulcanize#311 relating to `abspath` that seem to me to indicate they don't have that whole situation worked out yet. Further, it's possible that somebody expects to be able to provide a loader for their vulcanize instance, to do whatever odd task that allows them to access whatever `target.html`. Given `gulp-vulcanize` is centered on `gulp`, which already treats explicitly in local files, I think the minimum impact change is /probably/ to fix it here.

Finally, I ran into this because gulp expands paths out regardless, meaning I can't specify a relative path to the vulcanize instance if I wanted to. The easy workaround for me was to just copy the contents of this package to my codebase, and make the changes there, but wouldn't it be better to fix it in general? Maybe I'm the only person using gulp to vulcanize a file with absolute hrefs, but I don't think that's something that should be broken.